pictureflow/transform/object_detector.py

```diff
@@ -0,0 +1,56 @@
+from pictureflow.core import Image, Node
+
+import cv2
+
+
+class ObjectDetector(Node):
+
+    _input_type = Image
+    _output_type = Image
+
+    def __init__(self, parent, path_drop_threshold, tgt_size, id='detect_obj'):
+        super().__init__(parent, id)
+
+        self.drop_threshold = path_drop_threshold
+        self.tgt_size = tgt_size
+
+    def apply(self, item):
+        item.id = f'{item.id}-{self.id}'
+        img_raw = item.img_mat
+
+        gray = cv2.cvtColor(img_raw, cv2.COLOR_BGR2GRAY)
```

Comment:
It's still bugs to me that we need to use a CV color space conversion when we implemented a node for this ..